[Feat] Made getting started a little more clear (#35)
dema-trading-ai/wiki: docs/getting_started/running/running_python.md

# Running the Engine with Python

!!! note "" 

>     Running our engine with Python is not recommended. Only run it with Python when lacking knowledge of Docker. Please have a look [here](https://docs.dematrading.ai/getting_started/running/running_docker) for running it with Docker.

Reviewer: ```suggestion
    Running our engine with Python is not recommended. Only run it with Python if you are uncomfortable using Docker. Please have a look [here](https://docs.dematrading.ai/getting_started/running/running_docker) for running it with Docker.
```

Reviewer: Just a suggestion, sounds better to me, feel free to discard if you don't agree
